app/controllers/security_events_controller.rb

```diff
@@ -7,7 +7,6 @@ class SecurityEventsController < ApplicationController
   end
 
   def index
```

Comment:
it's not assumed! 

i am trying to standardize how our view-level authorization works before we start adding more roles. page-level stuff should be defined in the policy pages of the controller, so for this controller, it would be here: https://github.com/18F/identity-dashboard/blob/main/app/policies/security_event_policy.rb

as you can see below, i update the `index?` method to check for admin status `current_user&.admin?`. 

more details for pundit below: https://github.com/varvet/pundit

Reply:
ah, Rails is going to get me on these things until I get used to it. Thanks for explaining.